app/domain/queries/find_content.rb

```diff
@@ -9,8 +11,9 @@ def self.call(filter:)
   end
 
   def call
-    results = Aggregations::SearchLastThirtyDays.all
-                .joins('INNER JOIN dimensions_editions ON aggregations_search_last_thirty_days.dimensions_edition_id = dimensions_editions.id')
+    view = select_view
```

Comment:
I would isolate this behaviour in another class that can be tested in isolation.
What about something like this:

```ruby
 view_name = QueryPlanner.find_name(date_range)
```

This seems like something that don't belong to the finder; as a result we would also have less tests in the finder, and even the planner could be reused in the future.

Reply:
👍